src/test/java/org/prgrms/wumo/domain/location/api/LocationControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package org.prgrms.wumo.domain.location.api;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.prgrms.wumo.MysqlTestContainer;
+import org.prgrms.wumo.domain.location.dto.request.LocationRegisterRequest;
+import org.prgrms.wumo.domain.location.model.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
```

Comment:
Import 과정에서 ObjectMapper 주입이 잘 되지 않았다면 아래 클래스를 사용해보면 어떨까요???
```suggestion
import com.fasterxml.jackson.databind.ObjectMapper; 
```

Reply:
테스트코드 작성 당시에는 objectmapper가 빌드 에러를 일으키는 줄 알고 다른 클래스를 사용했는데 나머지 팀원분들이 문제없이 사용하는 것을 보니 저도 통일성을 위해 클래스를 교체하겠습니다!